apps/wfo-ui/pages/_app.tsx

```diff
@@ -56,35 +59,37 @@ function CustomApp({ Component, pageProps }: AppProps) {
     const [queryClient] = useState(() => new QueryClient(queryClientConfig));
 
     return (
-        <EuiProvider colorMode="light" modify={defaultOrchestratorTheme}>
-            <Head>
-                <title>Welcome to example-orchestrator-ui!</title>
-            </Head>
-            <main className="app">
-                <OrchestratorConfigProvider
-                    initialOrchestratorConfig={initialOrchestratorConfig}
-                >
-                    <QueryClientProvider
-                        client={queryClient}
-                        contextSharing={true}
+        <TranslationsProvider>
+            <EuiProvider colorMode="light" modify={defaultOrchestratorTheme}>
+                <Head>
+                    <title>Welcome to example-orchestrator-ui!</title>
```

Comment:
use translation

Reply:
The translation context is not available here yet. This requires a rearrangement of the the app page that I think is better saved for a separate PR once the translation approach is established. Or shall I just do it here right away?

Reply:
This _app is getting bigger and bigger with all the providers and contexts. Most of them should/might end up in the library to be consumed here. I would say to do fix this at that moment. I do agree that we eventually want to translate the title too.